### PR TITLE
Implement command .list areatriggers to list all areatriggers within the same map (if instanceable) or area (if continent) as the user

### PR DESCRIPTION
**Pull request by Nytanath**
## 🍰 Pullrequest

The command .list areatriggers shows every areatrigger in your same area (if you're in a continent) or map (if it's an instance or raid).

### Proof

- None

### Issues

- None

### How2Test

- .tele razorhill
- .list area (will show all areatriggers in Razor Hill)
- .tele zul'gurub
- Enter the raid.
- .list area (will show all areatriggers within the raid Zul'Gurub)

### CODE REVIEW

File: sql/updates/mangos/z2788_01_mangos_command.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+DELETE FROM `command` WHERE `name`="list areatriggers";
+INSERT INTO `command` (`name`, `security`, `help`) VALUES
+("list areatriggers", 3, "Syntax: .list areatriggers\n\nShow areatriggers within the same map (if inside an instanceable map) or area (if inside a continent) as the user.");
+
+DELETE FROM `mangos_string` WHERE `entry`=555;
+INSERT INTO `mangos_string` (`entry`, `content_default`) VALUES
+(555, "Showing all areatriggers in %s %s:");
```
Comment (killerwife) [inline sql/updates/mangos/z2788_01_mangos_command.sql:7]: Double quotes man in all of your commits.

Comment (Nytanath) [reply sql/updates/mangos/z2788_01_mangos_command.sql:7]: I thought double quotes in updates and single quotes in acid db? Sorry, it's hard to break a years old habit :P

Comment (killerwife) [reply sql/updates/mangos/z2788_01_mangos_command.sql:7]: Everything i know from sql is to never use double quotes ever. Sql standard tells to use single quotes and backticks.

Comment (killerwife) [reply sql/updates/mangos/z2788_01_mangos_command.sql:7]: https://stackoverflow.com/a/13089425

Comment (killerwife) [reply sql/updates/mangos/z2788_01_mangos_command.sql:7]: But to be fair we use backticks for object names

Comment (Nytanath) [reply sql/updates/mangos/z2788_01_mangos_command.sql:7]: Got it, thanks! Didn't know about the standard. I'll adapt.

**Pull request closed without merging**